ptx/S_12_flux_int.ptx

```diff
@@ -453,4 +454,103 @@
     </task>
   </activity>
 </subsection>
+<subsection>
+  <title>Efficently Calculating Flux Integrals</title>
+  <p>
+    In this section, we will look at some computational ideas that should allow you to more efficently compute the value of a flux integral. In many cases, the surface you are looking at the flux through can be written with one coordinate as a function of the others. For simplicity, let's consider <m>z=f(x,y)</m>.
+  </p>
+  <activity xml:id="function_flux_ints">
+    <title>Flux Integrals for Surfaces of the Form <m>z=f(x,y)</m></title>
+    <introduction>
+      <p>
+        In this activity, we will look at how to use a basic parameterization of a surface, <m>z=f(x,y)</m>, to efficently calculate flux integrals.
+      </p>
+    </introduction>
+    <task>
+      If a surface <m>S_1</m> is given by <m>z=f(x,y)</m>, use <m>s=x</m> and <m>t=y</m> to parameterize <m>S_1</m>.
+    </task>
+    <task>
+      Using your parameterization from the first part, show that the vector orthogonal to the surface <m>S_1</m> given by <m>\vr_s \times \vr_t</m> has the form
+      <me>
+        \vr_s \times \vr_t=\langle -\frac{\partial{f}}{\partial{x}},-\frac{\partial{f}}{\partial{y}},1 \rangle
+      </me>
+    </task>
+    <task>
+      <p>
+        For each of the three surfaces given below, 
+      <ul>
+        <li>Compute <m>\vr_s \times \vr_t</m> using the previous part</li>
+        <li>Graph your surface</li>
+        <li>Compute <m>\vr_s \times \vr_t</m> for 4 different points. You should make sure your vectors <m>\vr_s \times \vr_t</m> are orthogonal to your surface.</li>
+      </ul>
+      <ol>
+        <li><m>z=x^2+y^2</m></li>
+        <li><m>x+2y+z=-4</m></li>
+        <li><m>z=x^2-y^2</m></li>
+      </ol>
+      </p>
+    </task>
+    <task>
+      <p>
+        For each of the three surfaces,
+      <ol>
+        <li><m>z=x^2+y^2</m></li>
+        <li><m>x+2y+z=-4</m></li>
+        <li><m>z=x^2-y^2</m></li>
+      </ol>
+      use the calculations from <xref re="act_flux1">part</xref> and <xref ref="flux_int_calc">Theorem</xref> to compute the flux of each of the following vector fields through the part of the surface corresponding to the region <m>C</m>.
```

Comment:
@nelong The first `xref` here is broken when I compile. Can you fix that and push? If so, then I think we should be able to merge this and my PR this afternoon so week can keep moving forward with things.